metrics/config_observability.go

```diff
@@ -83,6 +87,16 @@ func NewObservabilityConfigFromConfigMap(configMap *corev1.ConfigMap) (*Observab
 		return nil, err
 	}
 
+	if raw, ok := configMap.Data["logging.enable-request-log"]; ok {
+		if strings.EqualFold(raw, "true") && oc.RequestLogTemplate != "" {
```

Comment:
Could you add unit tests for 1) flag set to true but template not set 2) flag not set but template set?

Reply:
sure